tensorboard/pip_package/setup.py

```diff
@@ -25,6 +25,8 @@
 
 
 REQUIRED_PACKAGES = [
+    'futures >= 3.1.1',
+    'grpcio >= 1.4.0',
```

Comment:
Please note that if we do this, it will become a transitive dependency of TensorFlow. Would that potentially cause problems?